### PR TITLE
feat(qwik-pod): Create test minor components

### PR DESCRIPTION
**Pull request by SDaian**
## Description

- [x] Creation of test for minor components

Closes: #1041

### CODE REVIEW

**Review by AllStackDev1 — APPROVED** [review]
image
So i get this errors...

**Review by AllStackDev1 — CHANGES_REQUESTED** [review]
Run test again some test are failing

File: qwik-graphql-tailwind/src/components/repo-tree/repo-tree.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    const { render } = await createDOM();
+
+    await render(<RepoTree />);
+  });
+
+  it(`should show the top repos`, async () => {
+    const { screen, render } = await createDOM();
+
+    await render(<RepoTree />);
+    console.log(screen.outerHTML);
```
Comment (iamjerryhogan) [inline qwik-graphql-tailwind/src/components/repo-tree/repo-tree.spec.tsx:47]: Please remove, and it seems this test isn't completed

File: qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        sortBy: '',
+      }),
+    };
+  });
+
+  it(`should mount`, async () => {
+    const { screen, render } = await createDOM();
+
+    await render(<UserDropdown username="thisDot" />);
+    console.log(screen.outerHTML);
```
Comment (iamjerryhogan) [inline qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.spec.tsx:23]: please remove and complete test

**Review by iamjerryhogan — CHANGES_REQUESTED** [review]
over five test failed please review.

**Review by markshenouda — APPROVED** [review]
Good job
image

**Pull request merged**